app/assets/stylesheets/application.css.scss

```diff
@@ -3,3 +3,4 @@
 @import "custom";
 
 
+
```

Comment:
How would you feel about removing some of these newlines at the end of this file?